.devcontainer/advanced-integration/welcome-message.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+WELCOME_MESSAGE="
+👋 Welcome to the \"Advanced Checkout Integration Example\"
+
+🛠️  Your environment is fully setup with all the required software.
+
+🚀 Once you rename the \".env.example\" file to \".env\" and update \"CLIENT_ID\" and \"APP_SECRET\", the checkout page will automatically open in the browser after the server is restarted."
+
+ALTERNATE_WELCOME_MESSAGE="
+👋 Welcome to the \"Advanced Checkout Integration Example\"
+
+🛠️  Your environment is fully setup with all the required software.
+
+🚀 The checkout page will automatically open in the browser after the server is started."
+
+if [ -n "$CLIENT_ID" ] && [ -n "$APP_SECRET" ]; then
```

Comment:
Do we have access to these environment variables in this bash script?

Reply:
Yes, CLIENT_ID, APP_SECRET  are environment variable can be accessed. These env variables are prompted  before launching the code space and user need to enter the values. But it is optional for the user to enter the values, if not it will display different message to user update the .env file etc.

Reply:
<img width="1046" alt="image" src="https://github.com/paypal-examples/docs-examples/assets/94279589/8aee25a5-bbd0-4929-8d3f-cfc68be53ca4">

Reply:
Yes, CLIENT_ID, APP_SECRET are environment variable can be accessed. These env variables are prompted before launching the code space and user need to enter the values. But it is optional for the user to enter the values, if not it will display different message to user update the .env file etc.